File: 2026/February-10-2026/running_sum.py
from typing import List
nums = [1,2,3,4]

def running_sum(nums: List[int]) -> List[int]: 
  prefix = [nums[0]]

  for i in range(1, len(nums)): 
    prefix.append(nums[i] + prefix[len(prefix) -1])
  return prefix

print(running_sum(nums))

File: 2026/February-10-2026/running_sum.js
nums = [1,2,3,4]

const running_sum = (nums) => { 
  prefix = [nums[0]]
  
  for(let i = 1; i < nums.length; i++){
    prefix.push(nums[i] + prefix[prefix.length -1])
  }
  return prefix
}

console.log(running_sum(nums))